naw/naw.py

```diff
@@ -65,3 +52,54 @@ def main():
             print(elem)
         if re.search(pattern, text, flags=re.IGNORECASE):
             print(text)
+
+
+def set_telephone(data, pattern):
```

Comment:
En data/pattern is net omgekeerd t.o.v. de pattern/data van print_phone number :-)